src/js/fs.js

```diff
@@ -20,6 +20,11 @@ var util = require('util');
 var fsBuiltin = native;
 
 fs.exists = function(path, callback) {
+  if ((!(util.isString(path)) && !(util.isBuffer(path)))
+      || (util.isFunction(path))) {
```

Comment:
ohh another thing: Do we need to explicitly check fo function here? Shouldn't the previous check be enough?

Reply:
Hmm, yes, you are definitely right, thanks for highlighting that.